### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by hwj0711** [mitroadmaps/roadtracer#42]
Can you share the prepared dataset?

**Comment by uakfdotb** [comment]
Code to prepare the dataset is in `https://github.com/mitroadmaps/roadtracer/tree/master/dataset`.

The prepared ground truth segmentation masks are available at https://roadmaps.csail.mit.edu/roadtracer/. The pre-trained model weights are also available there.

As we say there, the Google Maps imagery is copyrighted so we cannot redistribute it.
